### ISSUE TITLE
Installation error in Unbuntu 14.04 LTS with released 0.9.9 version

### ISSUE THREAD

**Issue by asw0316** [BinaryAnalysisPlatform/bap#373]
Here is the output I got when I did `make`

```
ocaml setup.ml -quiet -build 
File "_tags", line 25, characters 26-45:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 33, characters 26-46:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 36, characters 26-44:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 37, characters 26-48:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 49, characters 41-60:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 55, characters 41-61:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 58, characters 41-59:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 59, characters 41-63:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 60, characters 35-54:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 66, characters 35-55:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 69, characters 35-53:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 70, characters 35-57:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 73, characters 28-47:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 79, characters 28-48:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 83, characters 28-46:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 85, characters 28-50:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 90, characters 28-47:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 98, characters 28-48:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 102, characters 28-46:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 104, characters 28-50:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 120, characters 27-46:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 129, characters 27-47:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 133, characters 27-45:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 135, characters 27-49:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 141, characters 33-52:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 150, characters 33-53:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 154, characters 33-51:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 156, characters 33-55:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 173, characters 29-48:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 182, characters 29-49:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 186, characters 29-47:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 188, characters 29-51:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 195, characters 32-51:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 204, characters 32-52:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 208, characters 32-50:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 210, characters 32-54:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 217, characters 31-50:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 226, characters 31-51:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 230, characters 31-49:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 232, characters 31-53:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 241, characters 27-46:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 250, characters 27-47:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 254, characters 27-45:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 256, characters 27-49:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 272, characters 28-47:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 282, characters 28-48:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 288, characters 28-46:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 290, characters 28-50:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 305, characters 22-41:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 316, characters 22-42:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 322, characters 22-40:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 324, characters 22-44:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 339, characters 33-52:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 349, characters 33-53:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 356, characters 33-51:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 358, characters 33-55:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 374, characters 33-52:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 384, characters 33-53:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 391, characters 33-51:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 393, characters 33-55:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 408, characters 35-54:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 418, characters 35-55:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 425, characters 35-53:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 427, characters 35-57:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 442, characters 33-52:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 448, characters 33-53:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 453, characters 33-51:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 455, characters 33-55:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 461, characters 34-53:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 471, characters 34-54:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 478, characters 34-52:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 480, characters 34-56:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 495, characters 32-51:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 505, characters 32-52:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 512, characters 32-50:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 514, characters 32-54:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 531, characters 18-37:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 537, characters 18-38:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 543, characters 18-36:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 544, characters 18-40:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 548, characters 41-60:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 553, characters 41-55:
Warning: the tag "pkg_cohttp.lwt" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 560, characters 41-61:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 570, characters 41-59:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 573, characters 41-63:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 587, characters 25-44:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 592, characters 25-39:
Warning: the tag "pkg_cohttp.lwt" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 599, characters 25-45:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 609, characters 25-43:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 612, characters 25-47:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 627, characters 38-57:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 639, characters 38-58:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 646, characters 38-56:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 648, characters 38-60:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 663, characters 45-64:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 675, characters 45-65:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 682, characters 45-63:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 684, characters 45-67:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 699, characters 48-67:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 710, characters 48-68:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 717, characters 48-66:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 719, characters 48-70:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 732, characters 26-45:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 743, characters 26-46:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 750, characters 26-44:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 752, characters 26-48:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 766, characters 35-54:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 777, characters 35-55:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 786, characters 35-53:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 788, characters 35-57:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 802, characters 25-44:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 813, characters 25-45:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 822, characters 25-43:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 824, characters 25-47:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 839, characters 26-45:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 845, characters 26-46:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 849, characters 26-44:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 850, characters 26-48:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 851, characters 14-33:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 857, characters 14-34:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 861, characters 14-32:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 862, characters 14-36:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 864, characters 40-59:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 874, characters 40-60:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 881, characters 40-58:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 883, characters 40-62:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 903, characters 27-46:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 913, characters 27-47:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 920, characters 27-45:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 922, characters 27-49:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 943, characters 36-55:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 949, characters 36-44:
Warning: the tag "pkg_core" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 955, characters 36-56:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 961, characters 36-54:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 964, characters 36-58:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 978, characters 25-44:
Warning: the tag "pkg_bin_prot.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 984, characters 25-33:
Warning: the tag "pkg_core" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 990, characters 25-45:
Warning: the tag "pkg_fieldslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 996, characters 25-43:
Warning: the tag "pkg_sexplib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 999, characters 25-47:
Warning: the tag "pkg_variantslib.syntax" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
/home/antares-src/.opam/4.02.3/bin/ocamlfind ocamldep -package bitstring -syntax camlp4o -package bitstring.syntax -package camlp4 -package camlzip -syntax camlp4o -package comparelib.syntax -package compiler-libs -package core_kernel -package dynlink -syntax camlp4o -package enumerate.syntax -syntax camlp4o -package faillib.syntax -package fileutils -package findlib -syntax camlp4o -package herelib.syntax -package ocamlgraph -syntax camlp4o -package pa_ounit.syntax -package uuidm -package zarith -modules lib/bap/bap_plugin.ml > lib/bap/bap_plugin.ml.depends
+ /home/antares-src/.opam/4.02.3/bin/ocamlfind ocamldep -package bitstring -syntax camlp4o -package bitstring.syntax -package camlp4 -package camlzip -syntax camlp4o -package comparelib.syntax -package compiler-libs -package core_kernel -package dynlink -syntax camlp4o -package enumerate.syntax -syntax camlp4o -package faillib.syntax -package fileutils -package findlib -syntax camlp4o -package herelib.syntax -package ocamlgraph -syntax camlp4o -package pa_ounit.syntax -package uuidm -package zarith -modules lib/bap/bap_plugin.ml > lib/bap/bap_plugin.ml.depends
File "lib/bap/bap_plugin.ml", line 11, characters 7-13:
Failure: "Pa_type_conv: \"fields\" is not a supported type generator. (supported generators: compare, enumerate)"
File "lib/bap/bap_plugin.ml", line 1:
Error: Error while running external preprocessor
Command line: camlp4 '-I' '/home/antares-src/.opam/4.02.3/lib/ocaml/camlp4' '-I' '/home/antares-src/.opam/4.02.3/lib/bitstring' '-I' '/home/antares-src/.opam/4.02.3/lib/type_conv' '-I' '/home/antares-src/.opam/4.02.3/lib/comparelib' '-I' '/home/antares-src/.opam/4.02.3/lib/comparelib' '-I' '/home/antares-src/.opam/4.02.3/lib/enumerate' '-I' '/home/antares-src/.opam/4.02.3/lib/herelib' '-I' '/home/antares-src/.opam/4.02.3/lib/faillib' '-I' '/home/antares-src/.opam/4.02.3/lib/ocaml/compiler-libs' '-I' '/home/antares-src/.opam/4.02.3/lib/ocaml/compiler-libs' '-I' '/home/antares-src/.opam/4.02.3/lib/pa_ounit' '-parser' 'o' '-parser' 'op' '-printer' 'p' '-parser' 'o' '-parser' 'op' '-printer' 'p' 'unix.cma' 'bitstring.cma' 'bitstring_persistent.cma' 'pa_bitstring.cmo' 'pa_type_conv.cma' 'comparelib.cma' 'pa_compare.cma' 'enumerate.cma' 'pa_herelib.cma' 'faillib_syntax.cma' 'ocamlcommon.cma' 'pa_ounit_syntax.cma'  'lib/bap/bap_plugin.ml' > /tmp/ocamlppb42be4

Command exited with code 2.
make: *** [build] Error 1
```

How can I fix it? If you need more info for my environment, let me know.

Thanks!

**Comment by ivg** [comment]
Is this https://github.com/BinaryAnalysisPlatform/bap/issues/374#issuecomment-197901007 helps?

**Comment by asw0316** [comment]
It works! Thanks!
